src/components/Table/Internal/Body/Scroller.tsx

```diff
@@ -82,12 +87,13 @@ export const Scroller = React.forwardRef(
       const { height: stickyHeaderHeight = 0 } =
         scrollHeaderRef?.current?.getBoundingClientRect?.() || {};
       return (
-        rowTop -
-        scrollBodyTop +
-        stickyHeaderHeight +
-        rowHeight / 2 -
-        BUTTON_HEIGHT / 2 +
-        titleHeight
+        (rowTop -
+          scrollBodyTop +
+          stickyHeaderHeight +
+          rowHeight / 2 -
+          BUTTON_HEIGHT / 2 +
+          titleHeight) |
+        0
```

Comment:
this is one wild calculation

Reply:
Ya, needed to add the default too as upon first render there's a console warning of NaN, will look to improve this one.